### ISSUE TITLE
I don't install on Windows 2008, directory www\installer not found

### ISSUE THREAD

**Issue by Alexxandre3000** [cloverstudio/Spika-Server#88]
Hi,

I have two problems. No www/installer exists and Symfony reports (by composer)

  - Installing symfony/browser-kit (v2.4.1)
    Downloading: 100%

imagine/imagine suggests installing ext-gmagick (to use the Gmagick implementati
on)
imagine/imagine suggests installing ext-imagick (to use the Imagick implementati
on)
monolog/monolog suggests installing aws/aws-sdk-php (Allow sending log messages
to AWS services like DynamoDB)
monolog/monolog suggests installing doctrine/couchdb (Allow sending log messages
 to a CouchDB server)
monolog/monolog suggests installing ext-amqp (Allow sending log messages to an A
MQP server (1.0+ required))
monolog/monolog suggests installing ext-mongo (Allow sending log messages to a M
ongoDB server)
monolog/monolog suggests installing mlehner/gelf-php (Allow sending log messages
 to a GrayLog2 server)
monolog/monolog suggests installing raven/raven (Allow sending log messages to a
 Sentry server)
monolog/monolog suggests installing ruflin/elastica (Allow sending log messages
to an Elastic Search server)
symfony/routing suggests installing doctrine/annotations (For using the annotati
on loader)
symfony/routing suggests installing symfony/expression-language (For using expre
ssion matching)
symfony/event-dispatcher suggests installing symfony/dependency-injection ()
symfony/http-kernel suggests installing symfony/class-loader ()
symfony/http-kernel suggests installing symfony/dependency-injection ()
symfony/http-kernel suggests installing symfony/finder ()
silex/silex suggests installing symfony/css-selector (>=2.3,<2.5-dev)
silex/silex suggests installing symfony/form (>=2.3,<2.5-dev)
phpunit/php-code-coverage suggests installing ext-xdebug (>=2.0.5)
phpunit/phpunit suggests installing phpunit/php-invoker (>=1.1.0,<1.2.0)
symfony/dom-crawler suggests installing symfony/css-selector ()
symfony/browser-kit suggests installing symfony/process ()

**Comment by jaullo** [comment]
Hi @Alexxandre3000, what steps did u follow?
I have spika setup on windows 2008 R2 and is working without problems?

**Comment by Alexxandre3000** [comment]
No. Many problms. No have directory www/Installer and this related problem! :-(

**Comment by jaullo** [comment]
@Alexxandre3000  maybe i can send u a manual. If u wish!

**Comment by Alexxandre3000** [comment]
alexxandre3000@gmail.com thanks!

**Comment by superahmad** [comment]
@jaullo send me too abufaizmtx@gmail.com thanks

**Comment by jaullo** [comment]
Hello @Alexxandre3000  and @superahmad i gonna send the manual tomorrow at morning.

**Comment by tavox2025** [comment]
Send me please @Alexxandre3000 a manual tavox2025@gmail.com
